### DISCUSSION TITLE
api: get job execution log

### DISCUSSION THREAD

**Discussion by jgarec** [mend/renovate-ce-ee#394] [Ideas]
Hello,

When MEND_RNV_LOG_HISTORY_DIR is set, it would be interesting if we can get, at least, the last log file with a new api.
This is related to the other idea i proposed where we can have the jobs list.

I think it's easier with such api rather than consulting the log file on the FS. With this log file, we can see: 
* the status of the last execution
* the date
* ... and the detailed logs of course

BR

**Comment by Gabriel-Ladzaretti** [comment]
You can retrieve the stats of the last execution using the `/api/status`[^1] endpoint.

[^1]: https://github.com/mend/renovate-ce-ee/blob/main/docs/api.md#status
